StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -267,86 +275,71 @@ extension NativeFlowController {
             self.delegate?.authFlow(controller: self, didFinish: result)
         }
 
-        let completeFinancialConnectionsSession = { [weak self] in
-            guard let self = self else { return }
-            self.dataManager
-                .completeFinancialConnectionsSession(
-                    terminalError: customManualEntry ? "user_initiated_with_custom_manual_entry" : nil
-                )
-                .observe(on: .main) { [weak self] result in
-                    guard let self = self else { return }
-                    switch result {
-                    case .success(let session):
-                        let eventType = "object"
-                        if session.status == .cancelled
-                            && session.statusDetails?.cancelled?.reason == .customManualEntry
+        dataManager
```

Comment:
the key thing to double check in this code is that `showConfirmationAlert = false` should now be what the function executes